src/sentry/tasks/post_process.py

```diff
@@ -517,6 +520,10 @@ def post_process_group(
             # to ensure that we don't duplicate work should the forwarding consumers
             # need to rewind history.
             data = event_processing_store.get(cache_key)
+
+            if not data and options.get("rc-processing-split-write-double"):
+                data = transactions_processing_store.get(cache_key)
```

Comment:
you are going with a dual-write rollout strategy right? why do you need to do dual-read here as well?

Reply:
:+1: to Lyn's feedback. If you're going to double write, validate and the use. You shouldn't be reading from this store, only adding/deleting.